tpm/tpm.go

```diff
@@ -628,6 +631,125 @@ func MakeIdentity(rw io.ReadWriter, srkAuth []byte, ownerAuth []byte, aikAuth []
 	return blob, nil
 }
 
+func unloadTrspiCred(blob []byte) ([]byte, error) {
+	/*
+	 * Trousers expects the asym blob to have an additional data in the header.
+	 * The relevant data is duplicated in the TPM_SYMMETRIC_KEY struct so we parse
+	 * and throw the header away.
+	 * Trspi_UnloadBlob_UINT32(offset, &credSize, blob);
```

Comment:
I don't think this C code belongs in the comment. Instead, please explain this at a higher level